packages/components/src/internal/query/api.ts

```diff
@@ -751,6 +752,7 @@ export function updateRows(options: IUpdateRowsOptions): Promise<any> {
             auditBehavior: options.auditBehavior,
             auditUserComment: options.auditUserComment,
             form: options.form,
+            autoFormFileData: true,
```

Comment:
just curious on the reason for needing to pass this as `true` here instead of just always doing the "check for file values in the rows" on the api-js insertRows and updateRows case? I saw in that api-js PR that those two (insert/update) already tell the bind function that they support files, so isn't that enough?

Reply:
Good question. The `autoFormFileData` is the user-level configuration option. The bit you see being passed as `true` by `insertRows` and `updateRows` is those methods declaring that they do indeed `supportFiles` for binding where as others it doesn't make sense (e.g. `deleteRows`). So it requires a combination of `supportRows && autoFormFileData` to enable the row-level file processing.

Reply:
That makes sense, I get that the combination of those two booleans are what indicate the file processing / form data transformation. However, I thought we were hoping that the user / client caller code wouldn't need to know about this at all and could just call insertRows / updateRows with the file in their rows object and be good to go. From how I read this boolean prop usage, the caller would also need to know to pass true. Is that correct?